store/sql_channel_store.go

```diff
@@ -155,7 +155,7 @@ func (s SqlChannelStore) saveChannelT(transaction *gorp.Transaction, channel *mo
 		if count, err := transaction.SelectInt("SELECT COUNT(0) FROM Channels WHERE TeamId = :TeamId AND DeleteAt = 0 AND (Type = 'O' OR Type = 'P')", map[string]interface{}{"TeamId": channel.TeamId}); err != nil {
 			result.Err = model.NewLocAppError("SqlChannelStore.Save", "store.sql_channel.save_channel.current_count.app_error", nil, "teamId="+channel.TeamId+", "+err.Error())
 			return result
-		} else if count > 1000 {
+		} else if count > 20000 {
```

Comment:
It should be, we were trying to get it bumped before the release since some people were having issues.

Tix to make it configurable is at https://mattermost.atlassian.net/browse/PLT-4294